src/renderer/components/SearchInput/SearchInputButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import Icon from '../Icon'
+
+import styles from './styles.module.scss'
+
+import type { IconName } from '../Icon'
+
+type Props = {
+  icon: IconName
+  onClick: () => void
+  size?: number
+}
+
+export default function SearchInputButton({ icon, onClick, size = 20 }: Props) {
+  return (
+    <button className={styles.searchInputButton} onClick={onClick}>
```

Comment:
aria label for accessibility would be nice (not our current focus, but we will need to fix accessibility in the future)